sys/include/net/gnrc/ndp/internal.h

```diff
@@ -80,9 +80,12 @@ void gnrc_ndp_internal_send_nbr_sol(kernel_pid_t iface, ipv6_addr_t *tgt,
  *                          not be NULL.
  * @param[in] supply_tl2a   Add target link-layer address option to neighbor
  *                          advertisement if link-layer has addresses.
+ * @param[in] ext_opts      External options for the neighbor advertisement. Leave NULL for none.
```

Comment:
No it will not. To be honest: this functionality is just used at one part in the 6LoWPAN-ND (on handling NSs) and it's at the very end of the function. I just throw it in there and don't care for the result (because I don't use it afterwards). If you see a real use-case where we need a return value, I will a return value, otherwise I see it as a waste of resources (RAM, ROM and my time ;-P)